spec/integration/client/open_ssl_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe "openssl checks" do
+  let(:openssl_version_default) do
+    if windows? || aix?
+      "1.0.2zi"
+    elsif macos?
+      "1.1.1m"
+    else
+      "3.0.9"
```

Comment:
shouldn't this be 3.2.0

Reply:
@Stromweld no, 3.0.9 is the last FIPS-validated OpenSSL. The `openssl` gem's version is only loosely related.

Reply:
I see